packages/esm-patient-programs-app/src/programs/delete-program-modal.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { deleteProgramEnrollment, useEnrollments } from './programs.resource';
+import DeleteProgramModal from './delete-program.modal';
+import { showSnackbar, getCoreTranslation } from '@openmrs/esm-framework';
+
+jest.mock('./programs.resource', () => ({
+  deleteProgramEnrollment: jest.fn(),
+  useEnrollments: jest.fn(),
+}));
+jest.mock('@openmrs/esm-framework', () => ({
+  showSnackbar: jest.fn(),
+  getCoreTranslation: jest.fn((key, defaultText) => defaultText),
+}));
```

Comment:
Please see how to extend framework mocks - https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#you-should-not-need-to-use-partial-mocks.